src/spikeinterface/sorters/basesorter.py

```diff
@@ -262,7 +262,12 @@ def run_from_folder(cls, output_folder, raise_error, verbose):
             has_error = True
             run_time = None
             log["error"] = True
-            log["error_trace"] = traceback.format_exc()
+            error_log_to_display = traceback.format_exc()
+            trace_lines = error_log_to_display.strip().split("\n")
+            error_to_json = ["Traceback (most recent call last):"] + [
+                f"  {line}" if not line.startswith(" ") else line for line in trace_lines[1:]
```

Comment:
Is the space before `{line}` needed?

Reply:
I don't think so. Removing it.

Reply:
Oh no sorry I understand maybe it was to keep the left-alignment the same for lines that already start with a space vs. dont? In that case can leave, or maybe just `.strip()` all lines?

Reply:
I feel it does not matter that much? The big gain is dividing the error trace in lines so it does not appear like a super-long-line in json I feel.